giraffe/src/components/GeoLayer.tsx

```diff
@@ -0,0 +1,154 @@
+// Libraries
+import React, {Component} from 'react'
+import {AutoSizer} from 'react-virtualized'
+
+// Components
+import Geo from './Geo'
+
+// Types
+import {Config, Table} from '../types'
+import {GeoLayerConfig, GeoQueryVariables} from '../types/geo'
+
+interface OwnProps {
+  table: Table
+  config: GeoLayerConfig
+  plotConfig: Config
+}
+
+class GeoLayer extends Component<OwnProps> {
```

Comment:
Same here. Function Component with Hooks, please.

Reply:
It is done.